### PR TITLE
Add BIND documentation

### PR DESCRIPTION
**Pull request by mimugmail**

### CODE REVIEW

File: source/manual/how-tos/bind.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+BIND Plugin
+===========
+
+-------
+History
+-------
+
+The history of the Bind plugin was a user request on OPNsense subreddit to create a 
+plugin with a full-featured DNS server, also able to manage zonefiles with the most
+popular ressource records. In the beginning the plugin was build with only general
```
Comment (fichtner) [inline source/manual/how-tos/bind.rst]:
```suggestion
popular resource records. In the beginning the plugin was built with only general 
```

File: source/manual/how-tos/bind.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+-------
+History
+-------
+
+The history of the Bind plugin was a user request on OPNsense subreddit to create a 
+plugin with a full-featured DNS server, also able to manage zonefiles with the most
+popular ressource records. In the beginning the plugin was build with only general 
+features so the community can contribute and adding wished features with a friendly
+review of the OPNsense Team.
```
Comment (fichtner) [inline source/manual/how-tos/bind.rst]:
```suggestion
review of the OPNsense team.
```

File: source/manual/how-tos/bind.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    tries to resolve directly via the root servers.
+:Logsize in MB:
+    The amount for each logfile it can grow.
+:Maximum Cache Size:
+    This is the amount of RAM (in percent) the daemon can use for caching. 
+:Recursion:
+    You have to set a list of networks via **ACL** tab to allow them using recursion
+    against BIND.
+:DNSSec Validation:
+    Wheter to enable or disable DNSSec validation.
```
Comment (fichtner) [inline source/manual/how-tos/bind.rst]:
```suggestion
    Whether to enable or disable DNSSec validation. 
```

File: source/manual/how-tos/bind.rst [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+-----
+DNSBL
+-----
+
+:Enable DNSBL:
+    Enable the DNSBL service. BIND will be configured for Reverse Policy Zones to 
+    blacklist domains. Choose below the lists to use for blacklist categories.
+:Type of DNSBL:
+    Here you can select the lists to use. Do not just select all and save. There are
+    websites don't loading content when nested Ad's are not loaded.
```
Comment (fichtner) [inline source/manual/how-tos/bind.rst]:
```suggestion
    websites not loading content when nested ads are not loaded.
```

File: source/manual/how-tos/bind.rst [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+:Whitelist Domains:
+    When a website is blocked due to a false positiv you can enter the domain name here
+    so it get's whitelisted before the blacklists come into play.
```
Comment (fichtner) [inline source/manual/how-tos/bind.rst]:
```suggestion
    so it is whitelisted before the blacklists come into play.
```

**Comment by mimugmail** [comment]
Love them too 👍

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**